src/commands/General/Chat Bot Info/growth.ts

```diff
@@ -0,0 +1,55 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { ENABLE_INFLUX } from '#root/config';
+import { Mime } from '#utils/constants';
+import { fetch, FetchResultTypes } from '#utils/util';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Time } from '@sapphire/time-utilities';
+import { Message, MessageAttachment } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	cooldown: 10,
+	description: LanguageKeys.Commands.General.GrowthDescription,
+	extendedHelp: LanguageKeys.Commands.General.GrowthExtended,
+	permissions: ['ATTACH_FILES'],
+	enabled: ENABLE_INFLUX
+})
+export default class extends SkyraCommand {
+	private cachedGrowth: CachedImage = { cachedDate: Date.now(), buffer: null };
```

Comment:
Instead of `Date.now()`, it could store the next refresh time, so in line 20 you don't need to constantly add `Time.Hour * 12` all the time.